web/src/components/Diagram/components/DAG.styled.tsx

```diff
@@ -1,3 +1,4 @@
+import {Button} from 'antd';
```

Comment:
Hey @jorgeepc have you seen this https://reactflow.dev/docs/api/plugin-components/controls/ ?